docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
We should add a 'docs' tag to the parent (repo root) Makefile similar to [what EG does](https://github.com/jupyter/enterprise_gateway/blob/master/Makefile#L60-L61) - although you can ignore the source activation pre-command in elyra.

Reply:
Fixed.